sagemaker-pipelines/time_series_forecasting/amazon_forecast_pipeline/data/train.csv

```diff
@@ -0,0 +1,10887 @@
+datetime,season,holiday,workingday,weather,temp,atemp,humidity,windspeed,casual,registered,count
```

Comment:
What is the source of this data? Is it real or synthetic? I would like to host it in our public S3 bucket rather than committing it directly to the repository.

Reply:
It comes from https://github.com/aws-samples/amazon-forecast-samples/tree/main/notebooks/advanced/Incorporating_Related_Time_Series_dataset_to_your_Predictor/data here.